assets/js/googlesitekit/components-gm2/Button.js

```diff
@@ -122,7 +122,7 @@ const Button = forwardRef(
 			: null;
 
 		if (
-			( tooltip && tooltipTitle ) ||
+			( tooltip && tooltipTitle && ! disabled ) ||
 			( icon && tooltipTitle && children === undefined )
```

Comment:
Thanks for taking this approach - however, we want to avoid rendering the tooltip in all cases where the button is disabled, so the condition should be as follows. I'll apply the change myself to save the round trip.

```suggestion
			! disabled &&
			( ( tooltip && tooltipTitle ) ||
				( icon && tooltipTitle && children === undefined ) )
```

Reply:
Thanks!